src/components/Book.js

```diff
@@ -0,0 +1,23 @@
+const Book = () => {
+  return (
+    <article className='book'>
+      <div>
+        <h5>Action</h5>
+        <h3>The Hunger Games</h3>
+        <p className="book-info">Suzanne Collins</p>
```

Comment:
- [ ] [REQUIRED] Kindly make sure that this component takes the book data as a prop and renders the title and the author. The title and author should be dynamic not static